tests/Neo.Json.UnitTests/UT_JNumber.cs

```diff
@@ -26,20 +28,29 @@ public void SetUp()
         public void TestAsBoolean()
         {
             maxInt.AsBoolean().Should().BeTrue();
+            minInt.AsBoolean().Should().BeTrue();
             zero.AsBoolean().Should().BeFalse();
         }
 
+        [TestMethod]
+        public void TestBigInteger()
+        {
+            ((JNumber)BigInteger.One).AsNumber().Should().Be(1);
+            ((JNumber)BigInteger.Zero).AsNumber().Should().Be(0);
+            ((JNumber)BigInteger.MinusOne).AsNumber().Should().Be(-1);
+        }
```

Comment:
When doing testing, use `max` and `min` values please. You will see that using the `max` or `min` will fail.... `double` is to small to hold the data. `BigInteger` is 32 **BYTES** and `double` is 8 **BYTES** you get overflows. Its like doing
```csharp
var failed = checked((int)long.MaxValue);
```

![image](https://github.com/neo-project/neo/assets/8141309/eaf1a8ff-bf78-40d3-b6ab-b68c007e7dc0)

Reply:
ping @shargon

Reply:
Added ut with min and max